gradle/dependencies.gradle

```diff
@@ -45,7 +45,7 @@ ext {
       picasso                   : '2.71828',
       gmsLocation               : '16.0.0',
       ktlint                    : '0.34.2',
-      kotlinStdLib              : '1.3.70',
+      kotlinStdLib              : '1.3.71',
```

Comment:
`std lib` has been updated only, need to update `kotlin` version as well. Take a look at line [166 of this file](https://github.com/mapbox/mapbox-navigation-android/pull/2671/files#diff-6512f838e273b79676cac5f72456127fR166)
Also need to update `ktlint`, actually we should update it each time along with `kotlin` version, so may be should make a comment for this, wdyt?

Reply:
Great catch! I thought we were using only one for both. Fixing 👍

Reply:
Bumped `ktlint` to `0.36.0` although that's still including `1.3.60` 👀 https://github.com/pinterest/ktlint/releases/tag/0.36.0 PR updating to `1.3.70` hasn't been released yet https://github.com/pinterest/ktlint/pull/712